examples/helmcharts/nginx/values.yaml

```diff
@@ -195,9 +195,9 @@ ingress:
   ##
   ## It is also possible to create and manage the certificates outside of this helm chart
   ## Please see README.md for more information
-  # - name: example.local-tls
-  #   key:
-  #   certificate:
+  - name: example.local-tls
```

Comment:
Please comment out;  we shouldn't be editing the "examples".  They should be the raw charts from bitnami.

Reply:
I am trying to explain  before , I need this to catch if condition.